src/Components/folder/FolderCategoryItem.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import styled from "styled-components";
+import { useFolder } from "context/FolderContext";
+
+const DefaultFolder = styled.button`
+  padding: 8px 12px 8px 12px;
+  background-color: var(--color-white);
+  border: 1px solid var(--primary-color);
+  border-radius: 5px;
+  white-space: nowrap;
+
+  @media (max-width: 767px) {
+    padding: 6px 10px;
+    font-size: 14px;
+  }
+`;
+
+const SelectedFolder = styled(DefaultFolder)`
+  background-color: var(--primary-color);
+  color: white;
+`;
+
+const FolderCategoryItem = ({ id, children }) => {
+  const { folderFilter, setFolderFilter } = useFolder();
+
+  const onClickHandle = () => {
```

Comment:
@78-artilleryman 

```suggestion
  const handleFilterClick = () => ...
```

이벤트 핸들러의 경우 컨벤션에 따라 `handle[Action]` 을 따르는게 좋아보여요!